extensions/smallrye-openapi/deployment/src/main/java/io/quarkus/smallrye/openapi/deployment/SmallRyeOpenApiProcessor.java

```diff
@@ -331,6 +341,21 @@ LogCleanupFilterBuildItem logCleanup() {
                 "OpenAPI document initialized:");
     }
 
+    private void storeGeneratedSchema(byte[] schemaDocument, Format format) throws IOException {
+        Path directory = openApiConfig.storeSchemaDirectory.get();
+        if (!Files.exists(directory)) {
+            Files.createDirectories(directory);
```

Comment:
When specifying a relative path this appears to be relative to the working directory of where the build is running. I would instead have expected it to be relative to the Maven project's basedir. I will report an issue for this.

Reply:
I'll changes this. Thanks :)